spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/util/DeploymentPropertiesUtils.java

```diff
@@ -34,14 +39,16 @@
  */
 public final class DeploymentPropertiesUtils {
 
+	private static final Logger logger = LoggerFactory.getLogger(DeploymentPropertiesUtils.class);
+
 	private DeploymentPropertiesUtils() {
 		// prevent instantiation
 	}
 
 	/**
 	 * Pattern used for parsing a String of comma-delimited key=value pairs.
 	 */
-	private static final Pattern DEPLOYMENT_PROPERTIES_PATTERN = Pattern.compile(",\\s*app\\.[^\\.]+\\.[^=]+=");
+	private static final Pattern DEPLOYMENT_PROPERTIES_PATTERN = Pattern.compile(",\\s*(app|deployer)\\.[^\\.]+\\.[^=]+=");
```

Comment:
Yes, agree that we can solve this in a separate PR since this PR didn't cause the existing problem.